rubocop-salemove.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'rubocop-salemove'
-  s.version     = '0.0.3'
```

Comment:
Split version bump to a separate commit.

Reply:
b635e4